litmus-portal/frontend/src/containers/layouts/Header/ProfileDropdownItems.tsx

```diff
@@ -85,6 +83,22 @@ function ProfileInfoDropdownItems(props: ProfileInfoDropdownItemProps) {
     CallbackToSetSelectedProjectIDOnProfileDropdown(selectedProjectIDFromList);
   };
 
+  useEffect(() => {
+    const projectsAvailableForSwitching: Project[] = [];
+    projects.forEach((project) => {
+      const memberList: Member[] = project.members;
+      memberList.forEach((member) => {
+        if (
+          member.user_name === username &&
+          (member.invitation === 'Accepted' || member.invitation === '')
```

Comment:
The empty condition has to be removed

Reply:
okay @Sanjay1611 i am removing it then, we should merge the backend PR for auth first in that case @rajdas98

Reply:
PR #1982 @rajdas98 cc: @Sanjay1611